querybook/server/lib/table_upload/exporter/exporter_factory.py

```diff
@@ -1,27 +1,28 @@
-from typing import Dict, List
+from typing import Dict
 
 from app.db import with_session
 from logic.admin import get_query_engine_by_id
-from lib.utils.import_helper import import_modules
+from lib.utils.import_helper import import_module_with_default
 
 from .base_exporter import BaseTableUploadExporter
 
-ALL_TABLE_UPLOAD_EXPORTERS: List[BaseTableUploadExporter] = import_modules(
-    [
-        ("lib.table_upload.exporter.s3_exporter", "S3CSVExporter"),
-        ("lib.table_upload.exporter.s3_exporter", "S3ParquetExporter"),
```

Comment:
why removing these 2 exporters by default?

Reply:
see documentation that's added, you would need to add additional configs for use them